packages/forms/src/UIForm/UIForm.container.js

```diff
@@ -44,7 +44,7 @@ export default class UIForm extends React.Component {
 		if (!this.props.onChange) {
 			return;
 		} else if (this.props.moz) {
-			this.props.onChange(payload);
+			this.props.onChange(null, payload);
```

Comment:
Here you have the event (which can be null), just pass it. 
With that you can even remove this if/elseif/else by

```
if (this.props.onChange) {
    this.props.onChange(event, payload);
}
```